### PR TITLE
Add Java Cloud Hosting services

### PR DESCRIPTION
**Pull request by lazarenkod**
Added Cloud Hosting services for Java apps: AWS and Jelastic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,13 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JCTools](https://github.com/JCTools/JCTools) - Concurrency tools currently missing from the JDK.
 * [Koloboke](https://github.com/OpenHFT/Koloboke) - Hash sets and hash maps.
 
+## Hosting
+
+*Cloud hosting services for Java applications.*
```
Comment (jdandrews) [inline README.md]: I like this idea. You might consider adding AWS Lambdas as well.

Comment (jbduncan) [inline README.md]: You might also consider [Heroku](https://devcenter.heroku.com/categories/java).

I was initially also going to suggest adding [Google App Engine](https://cloud.google.com/appengine/), but it seems that it's support for Java 8 is in beta, so I hesitate to actually suggest it.

Comment (lazarenkod) [inline README.md]: Thank you for suggestion, I will add Heroku and AWS Lambda

File: README.md [hunk 2]
```diff
@@ -388,6 +388,13 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JCTools](https://github.com/JCTools/JCTools) - Concurrency tools currently missing from the JDK.
 * [Koloboke](https://github.com/OpenHFT/Koloboke) - Hash sets and hash maps.
 
+## Hosting
```
Comment (akullpp) [inline README.md]: Add the category to the table of contents.

File: README.md [hunk 3]
```diff
@@ -388,6 +388,13 @@ A curated list of awesome Java frameworks, libraries and software.
...
+
+*Cloud hosting services for Java applications.*
+
+* [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/) - AWS-based PaaS with support of Tomcat & Jetty.
+* [Jelastic](https://jelastic.com/jelastic-for-developers/) - Java cloud hosting(PaaS) with support of Tomcat, Jetty, GlassFish, JBoss, TomEE & WildFly.
```
Comment (akullpp) [inline README.md]: Please be as concise as possible. It should probably read: `Supports all major application servers.`

File: README.md [hunk 4]
```diff
@@ -388,6 +388,13 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JCTools](https://github.com/JCTools/JCTools) - Concurrency tools currently missing from the JDK.
 * [Koloboke](https://github.com/OpenHFT/Koloboke) - Hash sets and hash maps.
 
+## Hosting
+
+*Cloud hosting services for Java applications.*
+
+* [AWS Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk/) - AWS-based PaaS with support of Tomcat & Jetty.
```
Comment (akullpp) [inline README.md]: `Supports Tomcat and Jetty.`

**Comment by akullpp** [comment]
I assume that all hosting services are commercial products. So it would be nice if you could include  if they provide a free plan in the description.

**Comment by akullpp** [comment]
#416

**Pull request closed without merging**